rpcserver.go

```diff
@@ -1115,6 +1115,7 @@ func (r *rpcServer) GetInfo(ctx context.Context,
 		SyncedToChain:      isSynced,
 		Testnet:            activeNetParams.Params == &chaincfg.TestNet3Params,
 		Chains:             activeChains,
+		Alias:              r.server.currentNodeAnn.Alias.String(),
```

Comment:
This access is not thread safe. A way to get around this would be to call `genNodeAnnouncement(false)` which will return a copy of the announcement.